### PR TITLE
Common/string to hour

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/main/java/com/fundation/logic/controller/LoadSaveController.java [hunk 1]
```diff
@@ -20,8 +21,10 @@
 import com.fundation.logic.view.loadSaveCriteria.NameFrame;
 import com.fundation.logic.view.loadSaveCriteria.PopupLoadSave;
 
+import javax.swing.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/controller/LoadSaveController.java]: Use only needed classes

File: src/test/java/com/fundation/logic/common/StringToHourTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */package com.fundation.logic.common;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (aburgosu) [inline src/test/java/com/fundation/logic/common/StringToHourTest.java]: Import only used classes

File: src/main/java/com/fundation/logic/controller/ConvertController.java [hunk 3]
```diff
@@ -312,34 +316,79 @@ public void resetCriteria() {
...
+                            });
+                        } catch (Exception exception) {
+                            exception.getMessage();
                         }
                     }
                 }
             });
-        } catch (Exception exception) {
-            exception.getMessage();
+        } catch (Exception e) {
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/controller/ConvertController.java]: Rollback to exception variable name

File: src/test/java/com/fundation/logic/common/StringToHourTest.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * Verifies seconds result.
+     */
+    @Test
+    public void getSeconds_correctResult() {
+        String str = "12:34:51";
+        assertEquals(51, StringToHour.getSeconds(str));
+    }
+}
```
Comment (aburgosu) [inline src/test/java/com/fundation/logic/common/StringToHourTest.java]: Insert final break line

File: src/main/java/com/fundation/logic/controller/ConvertController.java (hunk 3 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/controller/ConvertController.java]: done

File: src/test/java/com/fundation/logic/common/StringToHourTest.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/test/java/com/fundation/logic/common/StringToHourTest.java]: done

File: src/test/java/com/fundation/logic/common/StringToHourTest.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/test/java/com/fundation/logic/common/StringToHourTest.java]: done

**Pull request merged**